### ISSUE TITLE
Missing version.txt file results in PHP warning

### ISSUE THREAD

**Issue by K14D** [TurbineCSS/Turbine#101]
To me txt-files in the root directory of an application seem to be of an sole informational purpose and typically aren't essential for script execution. With version.txt it's different. If it's missing, this will result in an PHP warning (depending on the error level). This should not be the case:

```

---
 css.php |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/css.php b/css.php
index d451eff..d4c78ee 100644
--- a/css.php
+++ b/css.php
@@ -22,7 +22,7 @@ $start = microtime(true);


 // Constants
-define('TURBINEVERSION', trim(file_get_contents('version.txt')));
+define('TURBINEVERSION', (file_exists('version.txt')) ? trim(file_get_contents('version.txt')) : 'unknown');
 define('TURBINEPATH', dirname($_SERVER['SCRIPT_NAME']));


-- 
```

**Comment by SirPepe** [comment]
Fix committed: 6649117eb17fd081ca0bf7d1b823ecb343f380fd

**Comment by SirPepe** [comment]
Fixed in 1.0.6
